velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -114,35 +125,50 @@ std::shared_ptr<DuckDbQueryRunner> ParquetTpchTest::duckDb_ = nullptr;
 std::shared_ptr<exec::test::TempDirectoryPath> ParquetTpchTest::tempDirectory_ =
     nullptr;
 TpchQueryBuilder ParquetTpchTest::tpchBuilder_(
-    facebook::velox::dwio::common::FileFormat::PARQUET);
+    dwio::common::FileFormat::PARQUET);
+
 std::unordered_map<std::string, std::string>
-    ParquetTpchTest::duckDbParquetWriteSQL_ = {std::make_pair(
-        "lineitem",
-        R"(COPY (SELECT l_orderkey, l_partkey, l_suppkey, l_linenumber,
+    ParquetTpchTest::duckDbParquetWriteSQL_ = {
+        std::make_pair(
+            "lineitem",
+            R"(COPY (SELECT l_orderkey, l_partkey, l_suppkey, l_linenumber,
         l_quantity::DOUBLE as l_quantity, l_extendedprice::DOUBLE as l_extendedprice, l_discount::DOUBLE as l_discount,
         l_tax::DOUBLE as l_tax, l_returnflag, l_linestatus, l_shipdate, l_commitdate, l_receiptdate,
-        l_shipinstruct, l_shipmode, l_comment FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {}))")};
-
-TEST_F(ParquetTpchTest, q1) {
-  auto tpchPlan = tpchBuilder_.getQueryPlan(1);
-  auto duckDbSql = duckDb_->getTpchQuery(1);
-  auto task = assertQuery(tpchPlan, duckDbSql);
+        l_shipinstruct, l_shipmode, l_comment FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {}))"),
+        std::make_pair(
+            "orders",
+            R"(COPY (SELECT o_orderkey, o_custkey, o_orderstatus,
+        o_totalprice::DOUBLE as o_totalprice,
+        o_orderdate, o_orderpriority, o_clerk, o_shippriority, o_comment
+        FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {}))"),
+        std::make_pair(
+            "customer",
+            R"(COPY (SELECT c_custkey, c_name, c_address, c_nationkey, c_phone,
+        c_acctbal::DOUBLE as c_acctbal, c_mktsegment, c_comment
+        FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {}))"),
+};
 
-  const auto& stats = task->taskStats();
-  // There should be two pipelines.
-  ASSERT_EQ(2, stats.pipelineStats.size());
-  // We used the default of 10 splits per file.
-  ASSERT_EQ(10, stats.numFinishedSplits);
+std::ostream& operator<<(std::ostream& os, const QueryParams& params) {
```

Comment:
consider moving this next to QueryParams structure for readability

Reply:
It can be outside, but still needs to be closer to the struct itself, e.g. right after the struct.